### ISSUE TITLE
Cannot init the evolution, threw "name 'watch' is not defined" and "KeyError: 'global'" if launch with -k param

### ISSUE THREAD

**Issue by ci-saya** [Gab0/japonicus#48]
When the program launched without duplicated gekko instance, the outputs were following,

```
The profits reported here are the profit beyond market price change;
	i.e. shown profit =  <backtest profit> - <market profit in evaluated candlestick period>;
> StochRSI
Evolving StochRSI strategy;

evaluated parameters ranges:
interval                      (7, 21)
thresholds.low                (15, 45)
thresholds.high               (45, 140)
thresholds.persistence        (4, 10)
Traceback (most recent call last):
  File "japonicus.py", line 99, in <module>
    gekko_generations(TargetParameters, GenerationMethod, EvaluationMode, web=APP)
  File "/root/Gekko/gekkoJaponicus/evolution_generations.py", line 77, in gekko_generations
    exchange_source=genconf.dataset_source)
  File "/root/Gekko/gekkoJaponicus/promoterz/evaluation/gekko.py", line 69, in getAvailableDataset
    raise "scanset not available: {}".format(watch)
NameError: name 'watch' is not defined
```
and it changed as following if it was with -k param:
```
Traceback (most recent call last):
  File "japonicus.py", line 35, in <module>
    settings['global']['gekkoPath']+'/web/server.js']
KeyError: 'global'
```

Using screen with a gekko instance running continuously.
Environment: clean Debian9 + nodejs 7.10.1 + python 3.6.0 (installed with pyenv)
requirements.txt fully installed with pip

A bit confused because the name 'watch' should be already in Settings.py which had it initially even without any personal conf.
Is it because I haven't completely changed the Settings.py and some params lack their proper values, or that it lacks some dependencies to be installed?

Thanks

**Comment by Gab0** [comment]
Hello @ci-saya!
Your first traceback suggests ur gekko install doest not have any candle databases available. If you specified exchange-currency on Settings.py we may have a problem, let me know what you set up there..
Second tb was indeed due to a bug hehe, so I'm submitting a fix..
Thanks for the report.

**Comment by ci-saya** [comment]
Hello @Gab0 sorry for reply lately,
for the first issue I took a look into Settings.py and found that the exchange was kraken and the asset was LTC whilst my Gekko is tracking BTC at bitfinex.
The program launched after I started and now it should work fine.
I created two files first output/evolution_gen_Locale1.csv and output/evolution_gen_Locale2.csv otherwise it would throw errors.

Anyway thank you very much! I will report if another issue would encounter.
